ipatests/test_integration/test_ntp_replacement.py

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+from ipaplatform.paths import paths
+
+
+class TestNTPreplacement(IntegrationTest):
```

Comment:
Why replacement :) configuration would be better. We are testing options functionality not some replacement.